db/change_cache.go

```diff
@@ -488,7 +488,7 @@ func (c *changeCache) DocChangedSynchronous(event sgbucket.FeedEvent) {
 
 		for _, seq := range syncData.RecentSequences {
 			if seq >= c.getNextSequence() && seq < currentSequence {
-				base.Infof(base.KeyCache, "Received deduplicated #%d for (%q / %q)", seq, base.UD(docID), syncData.CurrentRev)
+				base.Infof(base.KeyCache, "Received deduplicated #%d in _sync property for (%q / %q)", seq, base.UD(docID), syncData.CurrentRev)
```

Comment:
As above, arguably not directly related, but I think this adds valuable context for those who aren't intimately familiar with the background process here. Also, me, in about 48hrs.

Reply:
If we're changing this, I'd prefer to say something like 'in _sync.RecentSequences' here.

Reply:
Seems fair - I did consider using `KSyncXattrName` as well to make it nice and clean if we ever change (in a similar vein to the odd logging message that still references views). Might be overthinking it, but how much of that do we want hardcoded vs as constants?